staging/src/k8s.io/apimachinery/pkg/util/sets/BUILD

```diff
@@ -18,34 +11,7 @@ go_library(
         "string.go",
     ],
     importpath = "k8s.io/apimachinery/pkg/util/sets",
-)
-
-go_genrule(
```

Comment:
this rule was manually written (and there might be others), so we need something a bit smarter.